zuul.d/cisco-iosxr-jobs.yaml

```diff
@@ -84,15 +84,23 @@
     abstract: true
     vars:
       ansible_test_collections: true
-      ansible_test_skip_tags: local,network_cli
+      ansible_test_skip_tags: local,network_cli,iosxr_nonvoting
 
 - job:
     name: ansible-test-network-integration-iosxr-network_cli
     parent: ansible-test-network-integration-iosxr
     abstract: true
     vars:
       ansible_test_collections: true
-      ansible_test_skip_tags: local,netconf
+      ansible_test_skip_tags: local,netconf,iosxr_nonvoting
```

Comment:
same comment

Reply:
we will need this tag to skip those module which are failing with intermittent issue